evaluation/README.md

```diff
@@ -1,84 +1,95 @@
+
+### Text Classification
+
+
+| Transformation              | roberta-base-SST-2   | bert-base-uncased-QQP   | roberta-large-mnli   | roberta-base-imdb   |
+|:----------------------------|:---------------------|:------------------------|:---------------------|:--------------------|
+| BackTranslation             | 94.0->91.0 (-3.0)    | 92.0->90.0 (-2.0)       | 91.0->87.0 (-4.0)    | 95.0->92.0 (-3.0)   |
+| ButterFingersPerturbation   | 94.0->89.0 (-5.0)    | 92.0->89.0 (-3.0)       | 91.0->88.0 (-3.0)    | 95.0->93.0 (-2.0)   |
+| ChangePersonNamedEntities   | 94.0->94.0 (0.0)     | 92.0->92.0 (0.0)        | 91.0->89.0 (-2.0)    | 95.0->95.0 (0.0)    |
+| CloseHomophonesSwap         | 94.0->91.0 (-3.0)    | 92.0->88.0 (-4.0)       | 91.0->89.0 (-2.0)    | 95.0->96.0 (1.0)    |
+| DiscourseMarkerSubstitution | 94.0->94.0 (0.0)     | 92.0->92.0 (0.0)        | 91.0->91.0 (0.0)     | 95.0->95.0 (0.0)    |
+| MixedLanguagePerturbation   | 94.0->90.0 (-4.0)    | 92.0->86.0 (-6.0)       | 91.0->86.0 (-5.0)    | 95.0->91.0 (-4.0)   |
+| PunctuationWithRules        | 94.0->94.0 (0.0)     | 92.0->92.0 (0.0)        | 91.0->91.0 (0.0)     | 95.0->90.0 (-5.0)   |
+| ReplaceNumericalValues      | 94.0->94.0 (0.0)     | 92.0->92.0 (0.0)        | 91.0->90.0 (-1.0)    | 95.0->95.0 (0.0)    |
+| SentenceReordering          | 94.0->95.0 (1.0)     | 92.0->93.0 (1.0)        | nan                  | 95.0->94.0 (-1.0)   |
+
+
+Default models and datasets:
+
+- [SST-2](https://huggingface.co/datasets/glue): [textattack/roberta-base-SST-2](https://huggingface.co/textattack/roberta-base-SST-2)
+- [IMDB](https://huggingface.co/datasets/imdb): [textattack/roberta-base-imdb](https://huggingface.co/textattack/roberta-base-imdb)
+- [QQP](https://huggingface.co/datasets/glue): [textattack/bert-base-uncased-QQP](https://huggingface.co/textattack/bert-base-uncased-QQP)
+- [MNLI](https://huggingface.co/datasets/multi_nli): [roberta-large-mnli](https://huggingface.co/roberta-large-mnli)
+
+### Text-to-Text Generation
+### Text Tagging
+### Dialog Action to Text
+### Table-to-Text
+### RDF-to-Text
+### RDF-to-RDF
+### Question Answering
+
+| Transformation        |   deepset/roberta-base-squad2 |   bert-large-uncased-whole-word-masking-finetuned-squad |
+|:----------------------|------------------------------:|--------------------------------------------------------:|
+| RedundantContextForQa |                           5.6 |                                                    -1.9 |
+
+### Question Generation
+### AMR-to-Text
+### End-to-End Task
+
+### Paraphrase Detection
+
+| Transformation                   | paraphrase-xlm-r-multilingual-v1-PAWS |
+|:---------------------------------|:--------------------------------------|
+|                                  |                                       |

```

Comment:
Great work! Though, could you also update the numbers to the readme?

Reply:
Just made a commit with both available datasets and the results for LexicalCounterfactualGenerator which is the only transformation for Paraphrase Detection currently in the main branch. I have the results for the Paraphrase Detection transformations I have proposed, but I prefer to commit those as part of said transformations (once the evaluation method has been accepted). Cheers!